.github/workflows/cli-test-homebrew.yml

```diff
@@ -17,13 +17,17 @@ jobs:
         uses: actions/setup-node@v2
         with:
          node-version: ${{ matrix.node-version }}
+      - name: Check Out Repo
+        uses: actions/checkout@v2
       -  name: Global - Testing installation of Twilio CLI (1)
          run: |
-          brew tap twilio/brew && brew install twilio@${{ github.event.inputs.version }}
-          node test/asserts/validate_tests.js
+          brew tap twilio/brew && brew install twilio
+          chmod +x ./.github/scripts/cmds-to-test.sh
+          ./.github/scripts/cmds-to-test.sh
       - name: Global - Testing upgrade Twilio CLI (2)
         run: |
-          brew tap twilio/brew && brew install twilio@${{ github.event.inputs.version }}
+          brew tap twilio/brew && brew install twilio
```

Comment:
In this case it will install the latest CLI and do the upgrade, but shouldn't we install previous cli and upgrade to latest one?